components/Common/LanguageDropDown/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { LanguageIcon } from '@heroicons/react/24/outline';
+import * as DropdownMenu from '@radix-ui/react-dropdown-menu';
+import { useIntl } from 'react-intl';
+
+import { useLocale } from '@/hooks/useLocale';
+
+import styles from './index.module.css';
+
+const DropdownMenuDemo = () => {
+  const { availableLocales, currentLocale } = useLocale();
+  const intl = useIntl();
+
+  const ariaLabel = intl.formatMessage({
+    id: 'components.common.languageDropdown.label',
+  });
+
+  return (
+    <DropdownMenu.Root>
+      <DropdownMenu.Trigger asChild>
+        <button className={styles.iconWrapper} aria-label={ariaLabel}>
+          <LanguageIcon height="20" />
+        </button>
+      </DropdownMenu.Trigger>
+
+      <DropdownMenu.Portal>
+        <DropdownMenu.Content
+          align="start"
+          className={styles.dropDownContent}
+          sideOffset={5}
+        >
+          {availableLocales.map(({ name, code }) => (
+            <DropdownMenu.Item
+              key={name + code}
+              onClick={() => {}}
```

Comment:
Can you make it a prop?

```tsx
type LanguageDropDownProps = {
  onClick: () => void;
}

const LanguageDropDown: FC<LanguageDropDownProps> = ({ onClick = () => {} }) => {
  ...
}
```

Reply:
A click handler is required, but shouldn't it be an optional param if we're passing a default value?

Reply:
Hey, I updated the component to receive the prop.  Currently, it's optional cause we're supplying a default value but please let me know if this is not what's expected.

Reply:
Right, will re-review 🙇